frontend/server/src/Controllers/User.php

```diff
@@ -4605,6 +4606,68 @@ public static function apiRevokeAPIToken(\OmegaUp\Request $r) {
         \OmegaUp\DAO\APITokens::deleteByName($r->user->user_id, $name);
         return ['status' => 'ok'];
     }
+
+    /**
+     * @return array{entrypoint: string, templateProperties: array{payload: VerificationParentalTokenDetailsPayload, title: \OmegaUp\TranslationString}}
+     *
+     * @omegaup-request-param string $token
```

Comment:
The name of this param has changed, so you need update the declaration:
```suggestion
     * @omegaup-request-param string $parental_verification_token
```